### PR TITLE
mgr/volumes: convert string to bytes object

### PR DESCRIPTION
**Pull request by rishabh-d-dave**
Fixes: http://tracker.ceph.com/issues/39750

- [x] References tracker ticket
- [x] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

**Comment by rishabh-d-dave** [comment]
Can create subvolume with this test -

```
$ ./bin/ceph fs subvolume create a subvol1 2> /dev/null
$ ./bin/ceph fs subvolume create a subvol2 2> /dev/null
$ ./bin/ceph fs subvolume create a subvol3 2> /dev/null
$ ./bin/ceph fs subvolume rm a subvol1 2> /dev/null
$ ./bin/ceph fs subvolume rm a subvol2 2> /dev/null
$ ./bin/ceph fs subvolume rm a subvol3 2> /dev/null
```

**Comment by rishabh-d-dave** [comment]
Do we want a reproducer for a bug as minor as this too?

File: src/pybind/mgr/volumes/fs/subvolume.py [hunk 1]
```diff
@@ -95,7 +95,7 @@ def connect(self):
         log.debug("CephFS initializing...")
         self.fs.init()
         log.debug("CephFS mounting...")
-        self.fs.mount(filesystem_name=self.fs_name)
+        self.fs.mount(filesystem_name=self.fs_name.encode())
```
Comment (ajarr) [inline src/pybind/mgr/volumes/fs/subvolume.py]: s/self.fs_name.encode()/self.fs_name.encode('utf-8')/ ?

**Comment by ajarr** [comment]
@batrick , this looks good to merge. please take a look

**Pull request merged**